lib/ts/recipe/session/session.ts

```diff
@@ -125,6 +144,18 @@ export default class Session extends RecipeModule {
         return Session.getInstanceOrThrow().doesSessionExist();
     }
 
+    static addAxiosInterceptors(): boolean {
```

Comment:
This takes an `axios` instance. The function signature in `supertokens-website` is:
```
addAxiosInterceptors: (axiosInstance: any) => void
```